src/main/scala/Enemy.scala

```diff
@@ -0,0 +1,15 @@
+import scala.util.Random
+
+object Enemy {
+  val random: Random.type = scala.util.Random
+  var health: Int =  random.nextInt(20)
+  var attacks: Int =  random.nextInt(20)
+  var hit: Int = random.nextInt(10)
+  var initiative: Int =  random.nextInt(20)
+  def Reroll(){
```

Comment:
method names start with a lowercase word, only words after should have capitalised first letter